### PR TITLE
update the contributing doc with the committers information

### PR DESCRIPTION
**Pull request by bnmnetp**
So as not to break our own new rules, I even made a PR to update CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,6 +88,36 @@ Our Travis script will run jshint over all the source.  You should run jshint as
...
+generally people who have been around the project for a while and have "proven" themselves by contributing
+good code and ideas to the Skulpt.  This list may change over time as people gain or lose interest in
+Skulpt.  If you would like to volunteer contact Brad.
+
+Although a lot of our testing, and checking for adherence to the style guidelines is done automatically
+the review process is generally as follows:
+
+1.  Look at the diffs for each file on github, if it is pretty obvious what they are doing then great.
+2.  Look at the tests provided with the PR and try to think if there are additional tests that would provide better coverage. If you think additional tests are needed
+then you should let the owner of the PR know and have them add more tests.  I think that all new PRs that are adding new features
```
Comment (eah13) [inline CONTRIBUTING.md]: The 'I think that' can probably be removed for clarity.  Seems these guidelines are most helpful if they're opinionated about what's good vs bad.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -87,6 +88,36 @@ Our Travis script will run jshint over all the source.  You should run jshint as
...
+should be using the new unittest framework, not the old numbered framework.
+3.  Pull the the PR down to your local machine and run all the tests manually locally.  (Unless it is really trivial)
+4.  If the code looked a bit more complicated when you examined the diffs, then you should bring it up in your editor and look over the code in context and try to understand it better.  If there are 
+issues with how the code is written or if it is unclear about why something was done, then have that conversation with the owner of the PR.
+5.  Many of us have our own projects that exercise Skulpt to its limits. If you are particularly concerned about a PR then you may want to try out the built js files in your environment.
+6.  I think its always appropriate to raise questions and have a group conversation about anything that looks particularly problematic or risky.
+7.  With the new style unit tests, You should aks the submitter to file issues for tests that they comment out.  This will let us track completeness over time.
```
Comment (eah13) [inline CONTRIBUTING.md]: Great point.  Have we been doing this?  
Regardless, I do wonder if one issue per omission will get a little intense.  The list in #400 is already a bit of a monster.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -87,6 +88,36 @@ Our Travis script will run jshint over all the source.  You should run jshint as
...
+ 
+The current group of committers is as follows:
+
+* Brad Miller  bnmnetp
+* Scott Rixner rixner
+* Albert-Jan   albertjan
+* Meredydd Luff meredydd
+* Leszek Swirski  LeszekSwirski
```
Comment (eah13) [inline CONTRIBUTING.md]: I do believe @bzwheeler is on this list unless he's been removed at some point since the turtle stuff.

**Comment by eah13** [comment]
Excited to see this- I think it'll pay dividends in terms of our developer community for a long time. 

Dropped several non-blocking comments.  Please feel free to ignore and :ship: if they don't resonate and/or if others don't agree.

File: CONTRIBUTING.md (hunk 3 above)
Comment (bnmnetp) [inline CONTRIBUTING.md]: He does have write permission as a member of the feature branch team.  But I was waiting for him to confirm willingness to be a committer/reviewer more generally before adding him to this list.

**Comment by LeszekSwirski** [comment]
Small stylistic nit, the document now switches from "you" being the contributor, to "you" being the  committer. It's probably clear enough from the context who is who, but you may want to add a sentence along the lines of "if you are a committer, then..."

**Comment by bnmnetp** [comment]
A couple of other things I'm thinking about, now that I'm forced to actually think about it :smile:

* ``xxx.v` versus ``Sk.ffi.remapToJs(xxx)``   --  it seems like we should adopt a standard, and encourage people to follow it.
* return values on functions that do not return anything.  We ought to look for this and make sure the function is returning ``Sk.builtin.null.null$``
* Be on the lookout for functions that do not remap their parameters or return types.  I think this works in many fewer cases now than it used to, but in some cases where the function is mostly exercised internally but available externally this can slip through the cracks.

**Comment by eah13** [comment]
:+1: ready for merge to my eyes unless @LeszekSwirski wants to adjust style.  @LeszekSwirski: Those changes sound reasonable to me- you should be able to push them directly to `update-contrib` before merging if you want to clarify the writing.

**Comment by LeszekSwirski** [comment]
Merged, with the small stylistic fix of making the list of committers be links.

**Pull request merged**